meta-error-handling/static_assert.md

```diff
@@ -0,0 +1,113 @@
+# Module name: `static_assert`
+
+## Overview
+
+`static_assert` is a compile-time evaluated function that asserts the 
+truth of a supplied predicate, issuing an optional user-supplied error 
+message if the predicate is `false`.
+
+<table>
+  <thead>
+    <th>Level</th>
+    <th>Objectives</th>
+  </thead>
+  <tr>
+    <td>Foundational</td>
+    <td>Calling `static_assert` with a constant expression</td>
+  </tr>
+  <tr>
+    <td>Main</td>
+    <td>Using `static_assert` to detect contract violations and improve error messages</td>
```

Comment:
this doesnt show up as code-font in the table when I view it through github. Anyone aware of what I need to do in the markdown to get it as codefont?

Reply:
HTML: `<code>static_assert</code>`